AndroidManifest.xml

```diff
@@ -9,6 +9,28 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+
+    <!-- to start monitor services during boot -->
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+
+    <!-- to modify Seadroid's own account sync settings -->
+    <uses-permission android:name="android.permission.READ_SYNC_SETTINGS"/>
+    <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS"/>
+
+    <!-- get list of Seadroid accounts -->
+    <uses-permission android:name="android.permission.GET_ACCOUNTS"
+        android:maxSdkVersion="22"/>
```

Comment:
Why set the maxSDKVersion to 22? AFAIK, the latest API level is 23, is there any potential risk?